Eastern-Research-Group/mywaterway
# HMW-3 Compressed the widget background image.

## Related Issues:
* https://jira.epa.gov/browse/HMW-3

## Main Changes:
* Compressed the background image

## Steps To Test:
1. Navigate to http://localhost:9090/widgets/community-widgets.html
2. Verify the background image is visible and looks good
